cutscene_engine/cutscene_scripts/DialogueStarter.gd

```diff
@@ -0,0 +1,41 @@
+# Initializes the map's pawns and emits signals so the
+# Game node can start encounters
+# Processes in pause mode
+extends Node
+class_name DialogueStarter
+
+onready var dialogue_box = $DialogueInterface/DialogueBox
+#onready var trigger_box_setter = $TriggerBoxSetter
+
+var trigger_counter = 0
+
+export var key = ""
+var dialogue_key
+#func _ready() -> void:
+	#assert(dialogue_box)
+	#for action in get_tree().get_nodes_in_group("map_action"):
+		#(action as MapAction).initialize(self)
```

Comment:
dead code

Reply:
yeah forgot to remove this commented out code. It was there when the whole thing was being set up versus the old cutscene_engine from before